app/app.go

```diff
@@ -263,11 +262,8 @@ func NewApp(
 
 	app.keeper.upgrade = upgradekeeper.NewKeeper(skipUpgradeHeights, app.keys[upgradetypes.StoreKey], appCodec, homePath)
 
-	app.keeper.upgrade.SetUpgradeHandler("akashnet-2-upgrade-1", func(ctx sdk.Context, plan upgradetypes.Plan) {
-		migrations.MigrateAkashnet2Upgrade1(ctx, app.keeper.acct, app.keeper.bank, app.keeper.staking)
+	app.keeper.upgrade.SetUpgradeHandler("v0.42-upgrade", func(ctx sdk.Context, plan upgradetypes.Plan) {
 
-		// Set staking HistoricalEntries to 10000, required positive value for ibc
-		app.GetSubspace(stakingtypes.ModuleName).Set(ctx, stakingtypes.KeyHistoricalEntries, uint32(10000))
```

Comment:
Is there anything to upgrade?

Reply:
We don't have anything to upgrade here. Just we need a handler to upgrade to v0.42.4. We might need to add some migrator functions when upgrading to v0.43. 

We can keep this PR as draft until v0.43 release. Once after that, we can modify this handler and add migrator functions if needed any.